client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -146,6 +146,9 @@ export const VaccineCourseEditModal: FC<VaccineCourseEditModalProps> = ({
     }
   };
 
+  const isValid =
+    draft.name && !draft.vaccineCourseDoses?.some(dose => !dose.label);
```

Comment:
`.some()` will return true as soon as it finds an empty label so this is slightly more efficient than `.every()`

Reply:
Can we trim white space from name? Still can save blank name if we don't 😉